src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/AccessionVersionId.java

```diff
@@ -20,19 +20,13 @@
 import io.swagger.annotations.ApiModelProperty;
 
 import javax.persistence.Embeddable;
-import javax.validation.constraints.Min;
-import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Size;
 import java.io.Serializable;
 
 @Embeddable
 public class AccessionVersionId implements Serializable {
 
-    @Size(min = 1, max = 255)
-    @NotNull
     private String accession;
 
-    @Min(1)
```

Comment:
Since we are not going to control these at serialization time, could we do it at construction time, throwing IllegalArgumentException if the values don't fall in the acceptable ranges? One thing is having a null object, and the other constructing one with incorrect values.